portfolio/pom.xml

```diff
@@ -38,6 +38,14 @@
       <artifactId>jetty-annotations</artifactId>
       <version>${jetty.version}</version>
     </dependency>
+
```

Comment:
Consider removing extra space here to keep the code clean

Reply:
Okay. I will fix that. Thank you